core/tdk/src/utils/test_utils.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 Comcast Cable Communications Management, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use serde_json::json;
+
+pub trait Mockable {
+    fn mock() -> Self;
+}
+
+pub fn cap_jsonrpc_payload_granted(cap: String) -> serde_json::Value {
```

Comment:
In future we can expand this function to take vec of Role as well so that we can use this function to test for more test cases altering granted status for different roles. For now this is fine.